GenderPayGap.BusinessLogic/Repositories/UserRepository.cs

```diff
@@ -214,10 +287,14 @@ public async Task RetireUserAsync(User userToRetire)
             userToRetire.Modified = VirtualDateTime.Now;
 
             // save
-            await DataRepository.SaveChangesAsync();
+            await dataRepository.SaveChangesAsync();

```

Comment:
Here we're still awaiting the `SaveChangesAsync`. But further down we use the more preferred `SaveChangesAsync().Wait()`. Could we make these all work the same way?